src/block.ml

```diff
@@ -43,31 +40,31 @@ module Pre = struct
     let finish = finish link_defs in
     match next with
     | Rblockquote state ->
-        mk (Raw.Blockquote (finish state)) :: blocks
```

Comment:
I feel like the clarity gained by being able to remove this `mk` function thanks to the new Ast speaks the improvement here. :+1: